airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/spec.py

```diff
@@ -121,7 +123,7 @@ class Config:
         enum=["conversion", "impression", "mixed"],
     )
 
-    time_increment: Optional[PositiveInt] = Field(
+    time_increment: Optional[TimeIncrement] = Field(
```

Comment:
This will generate a oneOf in the spec which is not compatible with the platform. Hence, this would probably require this field to be the most generic type between the two types (between int and literal, literal is the most generic type) and this will require a config migration. 

I'll check just a bit more if there are other solutions we can come up with and come back to this comment before tomorrow.

Reply:
To help with the config migration, [here](https://github.com/airbytehq/airbyte/blob/c9065404c448b97f759eb4f717e2e887b8447b62/airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/config_migrations.py#L126-L193) is a good example of what it looks like. Basically, it should run at the beginning of any command that receives a config and it would do these things:
* Update the config file
* Emit a control message for the platform to update the config in DB